### ISSUE TITLE
Error in autogen.sh OSX 10.10 &.11 

### ISSUE THREAD

**Issue by JPaulMora** [tsiv/ccminer-cryptonight#9]
All deps installed from [home-brew](https://brew.sh) 

```
$ ./autogen.sh 
configure.ac:103: error: possibly undefined macro: AC_MSG_ERROR
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
```

**Comment by songproducer** [comment]
Try this:

brew install pkgconfig

**Comment by bigreddmachine** [comment]
I'm having the same problem and `brew install pkgconfig` does not fix the error. @JPaulMora did you have any luck figuring this out?

**Comment by JPaulMora** [comment]
No, same thing still happens. (re)installing pkgconfig doesn't fix but forgot to post. I might try building it on linux to see if is a Unix-specific problem or just OS X problem.

**Comment by metacollin** [comment]
OS X sometimes has trouble finding autoconf macros.  You need to put them in /usr/share/autoconf (which homebrew won't do automatically for you).  Alternatively, just put a copy in the same directory as your .ac files (the root of the repo in this case).  

If you want instant gratification, run this in the ccminer-cryptonight folder and then rerun autogen.sh.  Magic!

`curl https://source.jasig.org/cas-clients/mod_auth_cas/tags/mod_auth_cas-1.0.9.1/libcurl.m4 -o acinclude.m4`

**Comment by JPaulMora** [comment]
Hey! thanks! that actually helped.
